src/EditorFeatures/Next/Options/EditorConfigDocumentOptionsProvider.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.CodingConventions;
+
+namespace Microsoft.CodeAnalysis.Editor.Options
+{
+    // NOTE: this type depends on Dev15 assemblies, which is why the type is in EditorFeatures.Next. But, that library
+    // is rehostable and once we move .editorconfig support fully through the system, it should be moved to Workspaces
+    // or perhaps even lower.
```

Comment:
Laziness, since eventually such a split would go away once we finish the move of this down to the compiler/workspaces layer.